readthedocs/core/signals.py

```diff
@@ -62,4 +67,23 @@ def decide_if_cors(sender, request, **kwargs):  # pylint: disable=unused-argumen
 
     return False
 
+
+@receiver(pre_delete, sender=User)
+def delete_projects_and_organizations(instance, *args, **kwargs):
```

Comment:
The function arguments are wrong here. The first argument is `sender`, or the sending class (`User` in this case). I believe kwargs will have `instance` -- that is:

```python
@receiver(pre_delete, sender=User)
def delete_projects_and_organizations(sender, *args, **kwargs):
    instance = kwargs.pop('instance')
```

https://docs.djangoproject.com/en/1.9/ref/signals/#pre-delete

Reply:
Oh! Sorry, I missed the sender part. I will add it in next commit